### DISCUSSION TITLE
友達力拼轉型——關鍵10年後重返電子紙

### DISCUSSION THREAD

**Discussion by Yuhsin000** [1101-GS4719-C/Python-Community-GS4719-C#587] [閒聊八卦]
現在紙本閱讀模式逐漸沒落，電子紙越來越夯，目前元大做得不錯、友達也想跟進，不過兩者主攻方向不同，前者主攻室內而後者主攻戶外。
不知道大家對電子紙有沒有興趣 ( 聽說跟平板比起來比較不傷眼睛，也比較有紙本的感覺 )，以看書為目的時會比較想選擇平板還是電子紙呢？

https://udn.com/news/story/7240/6272585

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期待未來的發展。

**Comment by HsuChen88** [comment]
iPad如果有這種電子紙感覺會很棒

**Comment by ghost** [comment]
感覺現階段很不好評估主攻哪邊的優勢，畢竟有些人也很不太喜歡在室外環境閱讀，但其優點就是方便攜帶

**Comment by handsomeniko** [comment]
轉型成功的話 股票可以買起來

**Comment by d26488399** [comment]
我還是會選則平板，因為平板多工處理的能力比較強大，我可以用分割畫面一邊看題目一邊一邊寫，還有計算機的功能，這讓我可以躺在床上用一台裝置就寫完功課。

**Comment by bloodtrail0511** [comment]
感覺定位有點特殊，應用場景比較受限，目標客群會比較少

**Comment by Rosancl** [comment]
可以期待一下未來的發展，但跟平板的目標受眾區隔還是不小。

**Comment by Sazmafuji** [comment]
真 期待以後平板會如何發展

**Comment by yucheng1028** [comment]
ˊ之前鴻海好像也說過一樣的話，但好像不了了之

**Comment by cordydeng** [comment]
好厲害的技術，但我還是比較prefer實體書

**Comment by jcpan821** [comment]
感覺有戲

**Comment by Lai27330** [comment]
感覺單論電子書閱讀室還不錯的技術，但相比平板的多功能可能吸引力不太夠。

**Comment by HaoFish123** [comment]
感謝分享，以前沒怎麼聽說電子紙。

**Comment by treeann** [comment]
十年感覺是做得到的 加油

**Comment by gnsJhenJie** [comment]
版主想說的應該是「元太」不是「元大」?

**Comment by 110409538** [comment]
第一次聽說電子紙,好酷.

**Comment by sys511613** [comment]
這個技術好酷，期待未來的發展

**Comment by yucheng1028** [comment]
墊子只好酷啊

**Comment by stan890425** [comment]
這個相較傳統無法彎曲會有更多的用處

**Comment by bloodtrail0511** [comment]
第一次聽到電子紙欸，好酷

**Comment by etin2468** [comment]
如果要我選擇，我覺得還是色彩鮮豔的面板較吸引我，能看的東西比較多

**Comment by Lynn12378** [comment]
現在全面遠距一直看平板真的超傷眼，感覺用電子紙閱讀來減輕負擔很棒
